src/apps/engine/src/core_impl.cpp

```diff
@@ -252,6 +252,9 @@ void CoreImpl::ProcessEngineIniFile()
         Compiler->SetProgramDirectory(String);
     }
 
+    const auto use_cache = engine_ini->GetInt("script", "use_cache", false);
```

Comment:
[script] section is read and processed straight from COMPILER class. it makes sense to move it there as well

Reply:
done